pkg/vmprovider/providers/vsphere/session/session_vm_update.go

```diff
@@ -402,20 +402,29 @@ func UpdateHardwareConfigSpec(
 	configSpec *vimTypes.VirtualMachineConfigSpec,
 	vmClassSpec *v1alpha1.VirtualMachineClassSpec) {
 
-	// TODO: Looks like a different default annotation gets set by VC.
```

Comment:
I believe this is still the case

Reply:
Do you mean I should add the comment back?  The previous is maintained for non FSS enabled clusters because we are calling `UpdateConfigSpecAnnotations` method -- which will update the annotation to our default value from the one set by VC.